tasks/broad/GermlineVariantDiscovery.wdl

```diff
@@ -92,8 +92,15 @@ task HaplotypeCaller_GATK4_VCF {
     Boolean make_bamout
     Int preemptible_tries
     Int hc_scatter
-    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.1.8.0"
+    Boolean run_dragen_mode = false
```

Comment:
Same comment as before about defaulting once unless there is a reason to do otherwise

Reply:
Same question about backwards compatibility here